### ISSUE TITLE
[Windows] I can't use the local folder that I was using previously 

### ISSUE THREAD

**Issue by lmaestro** [owncloud/client#2757]
### Expected behavior
I can use whatever folder that I want in my new sync

### Actual behavior
When I try to use the same folder that I was using the last sync I have this error:
![screen shot 2015-01-29 at 9 37 32 am](https://cloud.githubusercontent.com/assets/4689430/5975626/35152ba0-a883-11e4-99d5-859957d9fa50.png)

### Steps to reproduce
1. Open desktop client and start to sync in the local folder "A"
2. Click on Modify account, introduce another account 
3. Select the local folder "A"

### Client configuration
Version 1.8.0-nightly20150129(build 4661)

Operating system: Windows 8.1

````
01-30 13:13:59:317 0x59c9ea8 :0 Setup local sync folder for new oC connection  "C:/Users/admin/ownCloud"
01-30 13:13:59:318 0x59c9ea8 :0 Setup-Log:  "Local sync folder C:/Users/admin/ownCloud already exists, setting it up for sync.<br/><br/>"
01-30 13:13:59:327 0x59c9ea8 :0 !!! OCC::EntityExistsJob created for  QUrl( "http://daily.owncloud.com/master/owncloud" )  querying "/"
01-30 13:13:59:935 0x59c9ea8 :0 void OCC::BandwidthManager::unregisterUploadDevice(OCC::UploadDevice*) OCC::UploadDevice(0x806a988)
01-30 13:14:00:273 0x59c9ea8 :0 ******** Remote folder found, all cool!
01-30 13:14:00:275 0x59c9ea8 :0 Setup-Log:  "A sync connection from C:/Users/admin/ownCloud to remote directory / was set up."
01-30 13:14:00:277 0x59c9ea8 :0 Setup-Log:  " "
01-30 13:14:00:278 0x59c9ea8 :0 Setup-Log:  "<p><font color="green"><b>Successfully connected to ownCloud!</b></font></p>"
01-30 13:14:00:284 0x59c9ea8 :0 Current Wizard page changed to  4
01-30 13:14:00:285 0x59c9ea8 :0 Setup-Log:  ""
01-30 13:14:00:289 0x59c9ea8 :0 Account was removed, removing associated folders: ("ownCloud")
01-30 13:14:00:289 0x59c9ea8 :0 folder  "ownCloud"  Terminating!
01-30 13:14:00:289 0x59c9ea8 :0 virtual void OCC::PropagateUploadFileQNAM::abort() OCC::PUTFileJob(0x9979b20) "#45/Clases SG - 20141028_190223 (2).pdf"
01-30 13:14:00:291 0x59c9ea8 :0 void OCC::AbstractNetworkJob::slotFinished() 5 "Operation canceled"
01-30 13:14:00:291 0x59c9ea8 :0 void OCC::PropagateUploadFileQNAM::slotPutFinished()  QUrl( "http://daily.owncloud.com/master/owncloud/remote.php/webdav/%2345/Clases SG - 20141028_190223 (2).pdf-chunking-439922108-5-0" )  FINISHED WITH STATUS 5 "Operation canceled" QVariant(Invalid) QVariant(Invalid)
01-30 13:14:00:292 0x59c9ea8 :0 QIODevice::read: device not open
01-30 13:14:00:293 0x59c9ea8 :0 ""
01-30 13:14:00:293 0x59c9ea8 :0 virtual void OCC::PropagateUploadFileQNAM::abort() OCC::PUTFileJob(0x99792b0) "#45/Clases SG - 20141028_190223 (2).pdf"
01-30 13:14:00:294 0x59c9ea8 :0 void OCC::AbstractNetworkJob::slotFinished() 5 "Operation canceled"
01-30 13:14:00:295 0x59c9ea8 :0 void OCC::PropagateUploadFileQNAM::slotPutFinished()  QUrl( "http://daily.owncloud.com/master/owncloud/remote.php/webdav/%2345/Clases SG - 20141028_190223 (2).pdf-chunking-439922108-5-1" )  FINISHED WITH STATUS 5 "Operation canceled" QVariant(Invalid) QVariant(Invalid)
01-30 13:14:00:297 0x59c9ea8 :0 virtual void OCC::PropagateUploadFileQNAM::abort() OCC::PUTFileJob(0x9d1dc20) "#45/Clases SG - 20141028_190223 (2).pdf"
01-30 13:14:00:299 0x59c9ea8 :0 void OCC::AbstractNetworkJob::slotFinished() 5 "Operation canceled"
01-30 13:14:00:300 0x59c9ea8 :0 void OCC::PropagateUploadFileQNAM::slotPutFinished()  QUrl( "http://daily.owncloud.com/master/owncloud/remote.php/webdav/%2345/Clases SG - 20141028_190223 (2).pdf-chunking-439922108-5-2" )  FINISHED WITH STATUS 5 "Operation canceled" QVariant(Invalid) QVariant(Invalid)
01-30 13:14:00:302 0x59c9ea8 :0 void OCC::SyncEngine::slotJobCompleted(const OCC::SyncFileItem&) "#45/Clases SG - 20141028_190223 (2).pdf" 3 "Operation canceled"
01-30 13:14:00:306 0x59c9ea8 :0 virtual void OCC::PropagateUploadFileQNAM::abort() OCC::PUTFileJob(0x99792b0) "#45/Clases SG - 20141028_190223 (2).pdf"
01-30 13:14:00:307 0x59c9ea8 :0 virtual void OCC::PropagateUploadFileQNAM::abort() OCC::PUTFileJob(0x9d1dc20) "#45/Clases SG - 20141028_190223 (2).pdf"
01-30 13:14:00:309 0x59c9ea8 :0 Removing  "ownCloud"
01-30 13:14:00:310 0x59c9ea8 :0 void OCC::SyncJournalDb::close() "C:/Users/admin/ownCloud/.csync_journal.db"
01-30 13:14:00:312 0x59c9ea8 :0 WRN: Failed to remove existing csync StateDB  "C:/Users/admin/ownCloud/.csync_journal.db"
01-30 13:14:00:313 0x59c9ea8 :0 Remove folder config file  "C:/Users/admin/AppData/Local/ownCloud/folders/ownCloud"
01-30 13:14:00:321 0x59c9ea8 :0 void OCC::SyncJournalDb::close() "C:/Users/admin/ownCloud/.csync_journal.db"
01-30 13:14:00:323 0x59c9ea8 :0 No database Transaction to commit
01-30 13:14:00:398 0x59c9ea8 :0 Saving  0  unknown certs.
01-30 13:14:00:400 0x59c9ea8 :0 Significant changes or first setup: switched to new account
01-30 13:14:00:402 0x59c9ea8 :0 "C:/Users/admin/AppData/Local/ownCloud//cookies.db"
01-30 13:14:00:413 0x59c9ea8 :0 Adding folder definition for "C:/Users/admin/ownCloud/" "/"
01-30 13:14:00:415 0x59c9ea8 :0 Could not remove old db file at "C:/Users/admin/ownCloud//.csync_journal.db"
01-30 13:14:00:421 0x59c9ea8 :0 Failed to resolve EGL function eglGetPlatformDisplayEXT (No se encontró el proceso especificado.)
01-30 13:14:00:425 0x59c9ea8 :0 static QWindowsEGLStaticContext* QWindowsEGLStaticContext::create(): Could not initialize EGL display: error 0x3001

01-30 13:14:00:426 0x59c9ea8 :0 static QWindowsEGLStaticContext* QWindowsEGLStaticContext::create(): When using ANGLE, check if d3dcompiler_4x.dll is available
01-30 13:14:06:525 0x59c9ea8 :0 !!! OCC::CheckServerJob created for  QUrl( "http://daily.owncloud.com/master/owncloud" )  querying "status.php"
01-30 13:14:06:758 0x59c9ea8 :0 Setup-Log:  "<font color="green"><b>Local sync folder C:/Users/admin/ownCloud/ successfully created!</b></font>"
01-30 13:14:06:761 0x59c9ea8 :0 * Setup folders from  "C:/Users/admin/AppData/Local/ownCloud/folders"
01-30 13:14:06:765 0x59c9ea8 :0 Set up  0  folders.
01-30 13:14:06:767 0x59c9ea8 :0 void OCC::ownCloudGui::slotShowSettings()
01-30 13:14:06:840 0x59c9ea8 :0 !!! OCC::CheckServerJob created for  QUrl( "http://daily.owncloud.com/master/owncloud" )  querying "status.php"
01-30 13:14:07:031 0x59c9ea8 :0 status.php returns:  QMap(("edition", QVariant(QString, "") ) ( "installed" ,  QVariant(bool, true) ) ( "maintenance" ,  QVariant(bool, false) ) ( "version" ,  QVariant(QString, "8.0.0.4") ) ( "versionstring" ,  QVariant(QString, "8.0 beta 2") ) )    0  Reply:  QNetworkReplyHttpImpl(0x9c40058)
01-30 13:14:07:034 0x59c9ea8 :0 ** Application: ownCloud found:   QUrl( "http://daily.owncloud.com/master/owncloud/status.php" )   with version  "8.0 beta 2" ( "8.0.0.4" )
01-30 13:14:07:036 0x59c9ea8 :0 # Check whether authenticated propfind works.
01-30 13:14:07:039 0x59c9ea8 :0 !!! OCC::PropfindJob created for  QUrl( "http://daily.owncloud.com/master/owncloud" )  querying "/"
01-30 13:14:07:358 0x59c9ea8 :0 status.php returns:  QMap(("edition", QVariant(QString, "") ) ( "installed" ,  QVariant(bool, true) ) ( "maintenance" ,  QVariant(bool, false) ) ( "version" ,  QVariant(QString, "8.0.0.4") ) ( "versionstring" ,  QVariant(QString, "8.0 beta 2") ) )    0 
````

**Comment by ckamm** [comment]
Relevant log entries:
```
01-30 13:14:00:309 0x59c9ea8 :0 Removing  "ownCloud"
01-30 13:14:00:310 0x59c9ea8 :0 void OCC::SyncJournalDb::close() "C:/Users/admin/ownCloud/.csync_journal.db"
01-30 13:14:00:312 0x59c9ea8 :0 WRN: Failed to remove existing csync StateDB  "C:/Users/admin/ownCloud/.csync_journal.db"
...
01-30 13:14:00:415 0x59c9ea8 :0 Could not remove old db file at "C:/Users/admin/ownCloud//.csync_journal.db"
```
Trying to reproduce...

**Comment by ckamm** [comment]
I can reproduce it on Win7.

**Comment by dragotin** [comment]
The reason might be that a sync is running in the background while the user tries to tweak the config. That's a duplicate, but I do not remember the other number.

**Comment by ckamm** [comment]
No, I don't see a running sync. I can also reproduce by removing the folder and readding it.

**Comment by ckamm** [comment]
It's not related to updating sqlite to 3.8.8.1 - same symptoms with that commit reverted.

**Comment by ckamm** [comment]
I started bisecting and this bug seems to be pretty old, but newer than 1.7.1-beta1. Details:
```
v1.7.1-beta1 good
cc3f7995fbced9dc1b941716d6e2b49278f99d1a bad
39097980b00b17b29e1467251908614763eaed34 bad
master bad
4d7b8bdb25b51128ab0a339ff946a9a5804b3a0a bad
421a8cc6b796f69e36cfc46d86e730676737f9a9 good
```

**Comment by lmaestro** [comment]
It is fixed in version 1.8.0-nightly20150202 (build 4670).
